docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<p>Maybe we could describe the two interfaces, either <code>.solve</code> or create a solver with all sorts of <code>.Sinkhorn()(...)</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>I feel like this is a bit too much for a tutorial. I can instead use the Sinkhorn class if this is a more recommended way to do ?</p><p> </p>